circle.yml

```diff
@@ -114,6 +96,10 @@ commands:
                             # do not run Docker in the interactive mode - adds control characters!
                             # and use Bash regex string comparison
                             command: |
+                                if [[ "$(node -p process.arch)" == 'arm64' ]]; then
+                                    echo "Currently cannot have Chrome on arm platform, skipping"
```

Comment:
```suggestion
                                    echo "Chrome does not support the arm platform (yet), skipping"
```

Reply:
I'd be useful for other devs to explain why we can't have Chrome, i.e. not supported, vs just saying we can't run it.

Reply:
5b3e66b4df535307fe9106bd7a8948d715d11f73